### ISSUE TITLE
Question - discrete 2-value legend

### ISSUE THREAD

**Issue by bblodfon** [jokergoo/ComplexHeatmap#256]
Hi,

I have matrix of 0 and 1's and I want to create a discrete legend. So, I do:
```
Heatmap(mat, ...,  heatmap_legend_param = list(at = c(1, 0), labels = c("YES", "NO"), 
          title = "Observed", title_position = "leftcenter-rot", direction = "vertical"))
```
and so I get the legend but it has the color bar from green to red, while I want two rectangular boxes, one green, the other red, with YES and NO respectively at their side. I searched through the documentation, but couldn't find something easy/straightforward. Could you help me with this?

BR, John.

**Comment by jokergoo** [comment]
You can add `color_bar = "discrete"` to `heatmap_legend_param`

**Comment by bblodfon** [comment]
Yes, thanks! (I saw it now in the documentation also, difficult to spot)
